packages/open-next/src/build.ts

```diff
@@ -47,9 +47,11 @@ export async function build(
   showWindowsWarning();
 
   // Load open-next.config.ts
-  const tempDir = initTempDir();
+  const tempConfigDir = fs.mkdtempSync(
```

Comment:
maybe we can rename this to `tempBuildDir` because it will become the `buildDir` (move)

Reply:
Yeah it make sense